packages/flutter_goldens_client/lib/client.dart

```diff
@@ -119,13 +119,39 @@ class GoldensClient {
   }
 
   Future<void> _checkCanSync() async {
+    final bool isCirrus = (platform.environment['CIRRUS_CI'] ?? '').isNotEmpty;
     final io.ProcessResult result = await process.run(
       <String>['git', 'status', '--porcelain'],
       workingDirectory: repositoryRoot.path,
     );
+    print('Initial check: ${result.stdout}');
```

Comment:
If you meant to leave this in, maybe we can give it a clearer label?

Reply:
I forgot to add the WIP label. ;)